### DISCUSSION TITLE
In order to prevent abuse, we’ve limited the number of invitations you can send in a day.  Try inviting new members tomorrow or contact support.

### DISCUSSION THREAD

**Discussion by noisyuu** [community/community#33677] [Other Feature Feedback, Questions, & Ideas]
As title says, I'm trying to add member to team of my university project, it always says "In order to prevent abuse, we’ve limited the number of invitations you can send in a day", but in fact I've never tried to add anyone for days. So please help me with issue, and this is captone project that is critical.

**Comment by liangjj21** [comment]
I encountered the same problem today, May I ask how it was resolved？
